openshift/template.yaml

```diff
@@ -64,12 +64,13 @@ objects:
     name: bug-master-service
   spec:
     ports:
-      - name: bug-master-port
-        port: ${{WEBSERVER_PORT}}
-        protocol: TCP
-        targetPort: ${{WEBSERVER_PORT}}
+    - name: bug-master-port
+      port: ${{WEBSERVER_PORT}}
+      protocol: TCP
+      targetPort: ${{WEBSERVER_PORT}}
     selector:
       app: bug-master
+    type: NodePort
```

Comment:
This wouldn't solve the issue, as nodes are not public. please look into [route](https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html)